src/loaders/xml/mod.rs

```diff
@@ -0,0 +1,39 @@
+mod client_info;
+
+use serde::Deserialize;
+use serde_xml_rs::de::Deserializer;
+use xml::reader::{EventReader, ParserConfig};
+
+pub use self::client_info::{ClientInfo, Connection};
+use super::GameFileLoader;
+#[cfg(feature = "debug")]
+use crate::debug::*;
+
+#[derive(Default)]
+pub struct XMLFileLoader;
+
+impl XMLFileLoader {
+    pub fn load_client_info(&self, game_file_loader: &mut GameFileLoader) -> ClientInfo {
+        let clientinfo = game_file_loader.get("data\\clientinfo.xml");
+        let sclientinfo = game_file_loader.get("data\\sclientinfo.xml");
+
+        let src = match (sclientinfo, clientinfo) {
+            (Ok(data), _) => String::from_utf8(data).unwrap(),
+            (_, Ok(data)) => String::from_utf8(data).unwrap(),
+            _ => panic!("FOO"),
+        };
+
+        // TODO: Make it work with euc-kr, since it panics
+        // with error: "Unsupported encoding: euc-kr"
+        let replaced_src = src.replace("euc-kr", "utf8");
```

Comment:
I tried to use the [`ParserConfig2` from `xml-rs`](https://github.com/netvl/xml-rs/blob/16348a470ac9e9903606925a7bf271c0627261e9/src/reader/config.rs#L198-L199) to override the file encoding when reading the `clientinfo.xml`, but it still fails with every available option.

I would keep it as-is until we find a solution for this.